js/main.js

```diff
@@ -50,21 +33,13 @@ const _minute = _second * 60;
 const _hour = _minute * 60;
 const _day = _hour * 24;
 
-// setInterval allows you to do certain action every period of time (like cronjob)
```

Comment:
If someone doesn't know what `setInterval` they can google it.  It's not the job of this file to teach you JS standard Library calls.

Where do we draw the line on comments like this? Why do we not have comments around stuff like `const`, `let`, `new Date()`